gluefactory/configs/pold2_mlp_dataloader_test.yaml

```diff
@@ -0,0 +1,24 @@
+data_dir: "pold2_mlp_dataset"
```

Comment:
Is there a reason why the data_dir is just the directory itself whereas the weights are given given as relative paths?

Reply:
Yeah, I thought over the course of testing the MLP we would keep exchanging weight files so better keep those as absolute paths and the data hopefully should not change a lot so kept it relative to the DATA_DIR of settings.py.